backend/app/api/endpoints/im_sessions.py

```
@@ -0,0 +1,99 @@
+)
+
+im_router = APIRouter()
+tasks_router = APIRouter()
+
+
+@im_router.get(
+    "/private-sessions",
+    response_model=IMPrivateSessionListResponse,
+)
+def list_private_sessions(
+    db: Session = Depends(get_db),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)